### DISCUSSION TITLE
LDAP Failed

### DISCUSSION THREAD

**Discussion by ismaelvillalba** [nilsteampassnet/TeamPass#3804] [General]
Hello! I'm having trouble syncing my domain's LDAP service with TeamPass 3.0.3. I am entering the configuration for its synchronization, but when I try to update the list of users, it shows as "In progress" and it never finishes loading.

Is there any guide or assistance available for the synchronization of this service? It is of the utmost importance. Thank you very much in advance.
![github](https://github.com/nilsteampassnet/TeamPass/assets/129912585/aca9a5a8-a3a0-451b-ba58-b107fab2991d)

**Comment by gschimon** [comment]
Hi!
I don't know, if this is still relevant, but I had the same problems with 3.0.10
After some debugging I found out, that the internal php-arrays for ldap are all lowercase. So for the LDAP-attributes, you also have to set lowercase values.
In my case (Microsoft Active Directory), I had to set the following attributes zu lowercase values:
- User Distinguished Name: distinguishedname
- User name attribute: samaccountname

**Comment by tooletime** [comment]
I have the same issue and all of my attributes are lower case.  Can you post your settings that you have working please?

**Comment by reginaldomoreno** [comment]
leave "Additional User DN" empty.
Don't use coma in "User Object Filter". You can use like this (objectCategory=Person)(memberOf=CN........)
